nym-data-observatory/src/config/data_observatory.rs

```diff
@@ -0,0 +1,20 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+pub struct DataObservatoryConfig {
+    pub webhooks: Vec<WebhookConfig>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct WebhookConfig {
+    pub id: String,
+    pub description: Option<String>,
+    pub webhook_url: String,
```

Comment:
would it make sense to use more strongly typed `Url` from the get go to avoid parsing (and possibly failing) later?